### PR TITLE
OLS-567: update_bundle_catalog.sh preserves relatedImage list in CSV

### PR DESCRIPTION
**Pull request by raptorsun**
## Description

update update_bundle_catalog.sh script to keep relatedImage list in CSV, update default bundle image URL, tags, etc.

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [x] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue # [OLS-567](https://issues.redhat.com//browse/OLS-567)
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -314,6 +322,23 @@ OPM = $(shell which opm)
...
 
+.PHONY: yq
+YQ = ./bin/yq
+yq: ## Download yq locally if necessary.
+ifeq (,$(wildcard $(YQ)))
+ifeq (,$(shell which yq 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(YQ)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
```
Comment (tisnik) [inline Makefile:333]: why is it needed? you are downloading binary to be run, so can be put anywhere on PATH, right?

File: hack/update_bundle_catalog.sh [hunk 2]
```diff
@@ -7,64 +7,102 @@
...
   echo "opm version v1.39.0 is required"
   exit 1
 fi
 
-TAG="latest"  # Set the operator version
-BUNDLE_TAG="0.0.1"    # Set the bundle version
+: ${YQ:=$(which yq)}
+echo "using yq from ${YQ}"
+# check if yq exists
+if [ -z "${YQ}" ]; then
```
Comment (tisnik) [inline hack/update_bundle_catalog.sh:21]: `command -v yq` will check if it's really executable, might be more precise test

**Review by tisnik — COMMENTED** [review]
it looks ok, just two nitpicks

File: Makefile (hunk 1 above)
Comment (raptorsun) [reply Makefile:333]: yq is for manipulating the CSV YAML file. 
it is downloaded to `./bin` as other tools do.

**Pull request merged**